### PR TITLE
chore(migrate repository): migrate repository from ng to tractus-x

### PR DESCRIPTION
**Pull request by stephanbcbauer**
## Description

This PR add content from a repostory [product-ontology](https://github.com/catenax-ng/product-ontology) which was managed in a different organisation, [catenax-ng](https://github.com/catenax-ng) including the history
 
Since we need this content in tractus-x and the catenax-ng will be deleted soon, we want to migrate the content to tractus-x.

Therefore, this initial [PR](https://github.com/eclipse-tractusx/sldt-ontology-model/pull/2) was created.

It contains all the content and history of the repository.

A review [Eclipse IP Ticket](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/15625) was already approved with the current content, but it didn't include new legal docs, header a.s.o.

**So in this case we will create a new review request, after we are finished with our work, before the PR is merged**

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (evegufy) [inline .github/PULL_REQUEST_TEMPLATE.md:2]: wrong header, applies to all files

**Review by evegufy — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (stephanbcbauer) [reply .github/PULL_REQUEST_TEMPLATE.md:2]: Yes, if I adapt the PR right now (or the content) which was already approved by EF, I guess I would have to ask again for approval.

We have to adapt Copyrights, legal documents, Workflows anyway. This was planned on Monday directly on the repository after initial merge.

Comment (SebastianBezold) [reply .github/PULL_REQUEST_TEMPLATE.md:2]: It was wrong to open the IP issue with these headers in. 
The approval is also wrong in my opinion. Maybe the automated check missed it, but these headers definitely need to be reworked

Comment (stephanbcbauer) [reply .github/PULL_REQUEST_TEMPLATE.md:2]: We will adapt it and ask for a new review.

**Comment by stephanbcbauer** [comment]
Converted to DRAFT … after the Header, and legal docs are updated … @jSchuetz88 and I will change the status again.

**Comment by AngelikaWittek** [comment]
@stephanbcbauer 
Also do the changes from: https://github.com/eclipse-tractusx/sldt-ontology-model

- NOTICE file is the wrong template
- CONTRIBUTING file -> update header, references to Catena-X, use new template
- CODE of Conduct -> Use new template

**Comment by stephanbcbauer** [comment]
> @stephanbcbauer Also do the changes from: https://github.com/eclipse-tractusx/sldt-ontology-model
> 
> * NOTICE file is the wrong template
> * CONTRIBUTING file -> update header, references to Catena-X, use new template
> * CODE of Conduct -> Use new template

Thanks @AngelikaWittek , @jSchuetz88  will update the repository as soon as possible.

**Comment by arnoweiss** [comment]
Can you briefly state the reasoning for transfering governance of this repository (and thus implicitly the `https://w3id.org/catenax` namespace) to Eclipse Tractus-X? I've always thought of this as very common to the policy constrains - all of which are managed in the catenax-ev organization.

**Comment by stephanbcbauer** [comment]
> Can you briefly state the reasoning for transfering governance of this repository (and thus implicitly the `https://w3id.org/catenax` namespace) to Eclipse Tractus-X? I've always thought of this as very common to the policy constrains - all of which are managed in the catenax-ev organization.
> 
> cc @obalandi

@jSchuetz88

**Comment by stephanbcbauer** [comment]
@obalandi @jSchuetz88 dont want to stress anybody, but i think we should finalize this PR?

What's missing?
- Answering the last questions @obalandi 
- review by committees
- create new IP Ticket @stephanbcbauer 

After these steps, i think we are good to merge the big PR and we are able to delete the fork.

THX

**Comment by stephanbcbauer** [comment]
> @stephanbcbauer, we (@jSchuetz88 and @obalandi) discussed this with @arnoweiss. And the decision is that all vocabularies in the taxonomy that are used in an application configuration will have a special attribute so that they are protected and will not be changed in a normal modelling session. How this works will be defined in the modelling guidelines in TC4M.

OK, thank you very much. So in this case, we are ready for review by committers, correct? Meanwhile, I will create the Eclipse IP Ticket, cause of the big number of changes.

**Comment by stephanbcbauer** [comment]
> > @stephanbcbauer, we (@jSchuetz88 and @obalandi) discussed this with @arnoweiss. And the decision is that all vocabularies in the taxonomy that are used in an application configuration will have a special attribute so that they are protected and will not be changed in a normal modelling session. How this works will be defined in the modelling guidelines in TC4M.
> 
> 
> 
> OK, thank you very much. So in this case, we are ready for review by committers, correct? Meanwhile, I will create the [Eclipse IP Ticket](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/16442), cause of the big number of changes.

@obalandi @jSchuetz88 ip ticket was approved.

**Review by arnoweiss — APPROVED** [review]
I think we should get the migration over with and leave bugfixes resulting from the migration to the semantic modeling expert group.

**Review by matbmoser — APPROVED** [review]
IF ip check was approved, looks good to me, please include the trufflehog workflow too after merged. Issue #3

**Comment by stephanbcbauer** [comment]
@evegufy , fine for you? I think your requested changes are done, or obsolete. Thx after your approval I will merge it

**Review by evegufy — APPROVED** [review]

**Comment by stephanbcbauer** [comment]
Thank you, everybody, for your effort. PR is merged, I will also delete the related fork.

**Pull request merged**
